Milvus.Client/FieldSchema.cs

```diff
@@ -95,6 +95,42 @@ public static FieldSchema CreateBinaryVector(string name, int dimension, string
     public static FieldSchema CreateJson(string name)
         => new(name, MilvusDataType.Json);
 
+    /// <summary>
+    /// Create a field schema for a <c>array</c> of <c>TData</c> field.
+    /// </summary>
+    /// <typeparam name="TData">Determines the element data type of array stored in the field based.</typeparam>
+    /// <param name="name">The field name.</param>
+    /// <param name="capacity">Maximum number of elements that an array field can contain.</param>
+    /// <param name="description">An optional description for the field.</param>
+    public static FieldSchema CreateArray<TData>(
+        string name,
+        int capacity,
```

Comment:
Maybe `maxCapacity` as the parameter is actually called in Milvus?

Reply:
Renamed